src/components/Pagination.jsx

```diff
@@ -15,12 +16,12 @@ export const Pagination = ({
   listOfItems,
   setCurrentPageItems,
 }) => {
-  const [activePageIDState, setActivePageIDState] = useState(1);
+  const [currentPageState, setCurrentPageState] = useState(1);
 
   const totalPages = useMemo(() => {
-    return calculateNumberOfPages(listOfItems);
+    return calculateTotalPages(listOfItems);
 
-    function calculateNumberOfPages(list) {
+    function calculateTotalPages(list) {
       if (list.length > 0) {
         const totalPages = Math.ceil(list.length / itemsPerPage);
         return totalPages;
```

Comment:
This function declaration doesn't look to be needed, since you're only using it inside this `useMemo` block. You could just return the calculated value directly from the block.